### PR TITLE
[DOCS] Improve DataGrip Document

### PR DESCRIPTION
**Pull request by zwangsheng**
zwangsheng [Closes #427](https://github.com/yaooqinn/kyuubi/pull/427)    Target Issue [&#10088;?&#10089;](https://pullrequestbadge.com/?utm_medium=github&utm_source=yaooqinn&utm_campaign=badge_info)

### _Why are the changes needed?_

Fix  the repeat serial number of quick start with dataGrip

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.readthedocs.io/en/latest/tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by yaooqinn** [comment]
please resolve the conflicts

**Comment by zwangsheng** [comment]
> please resolve the conflicts

ok, i reslove the conflicts

File: docs/quick_start/quick_start_with_datagrip.md [hunk 1]
```diff
@@ -1,41 +1,41 @@
-# 5.Getting Started With Kyuubi and DataGrip
-## 5.1 What is DataGrip
+# Getting Started With Kyuubi and DataGrip
+## What is DataGrip
 DataGrip is a multi-engine database environment released by JetBrains, supporting MySQL and PostgreSQL, Microsoft SQL Server and Oracle, Sybase, DB2, SQLite, HyperSQL, Apache Derby, and H2.
 
 [About DataGrip](https://www.jetbrains.com/datagrip/)
```
Comment (pan3793) [inline docs/quick_start/quick_start_with_datagrip.md:5]: Can we remove `[About DataGrip]` and just add hyperlink to the first word `DataGrip`?

Comment (zwangsheng) [reply docs/quick_start/quick_start_with_datagrip.md:5]: ok， i will resolve

**Comment by zwangsheng** [comment]
i will close this pull request, And make an new one with another branch

File: docs/quick_start/quick_start_with_datagrip.md [hunk 2]
```diff
@@ -1,41 +1,41 @@
...
+## Interacting With Kyuubi Server
 Now, you can interact with Kyuubi server.
 
 The left side of the photo is the table, and the right side of the photo is the console.
 
 You can interact through the visual interface or code.
 
 ![workspace](../imgs/dataGrip/workspace.png)
-## 5.5 At The End
+## At The End
```
Comment (yaooqinn) [inline docs/quick_start/quick_start_with_datagrip.md:40]: The End looks better
